synthetic/derivatives/fmriprep/dataset_description.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Name": "fMRIPrep - fMRI PREProcessing workflow",
+    "BIDSVersion": "1.6.0",
+    "DatasetType": "derivative",
+    "GeneratedBy": [
+        {
+            "Name": "fMRIPrep",
+            "Version": "1.0.6",
+            "CodeURL": "https://github.com/nipreps/fmriprep/archive/refs/tags/1.0.6.tar.gz"
```

Comment:
yup I could try to have something that matches a more recent output of fmriprep but see this discussion: https://github.com/bids-standard/bids-examples/pull/293#discussion_r726204897

Reply:
@effigies 

tempted to leave this as is:

- the old version of fMRIprep would not give this type of output
- a newer version of fMRIprep would not give this output either (AFAICT)
- also this can be used as an example of how to use `GeneratedBy` and CHANGES to keep track of how changes came about

Reply:
Yup the dataset description was adjusted to mention the manual change. Will resolve this then.